packages/playground/src/weblets/profile_manager.vue

```diff
@@ -155,6 +155,8 @@
                               :disabled="creatingAccount || activatingAccount || activating"
                               @click:append="reloadValidation"
                               readonly
+                              @input="getEmail"
+                              :loading="loadEmail"
```

Comment:
check if `mnemonic` is valid before loading Email otherwise it will throw an error